spree_gateway.gemspec

```diff
@@ -24,6 +24,7 @@ Gem::Specification.new do |s|
   spree_version = '>= 3.1.0', '< 5.0'
   s.add_dependency 'spree_core', spree_version
   s.add_dependency 'spree_extension'
+  s.add_dependency 'deface'
```

Comment:
we also need to require deface here: https://github.com/spree/spree_gateway/blob/master/lib/spree_gateway.rb